Modules/FT0/include/FT0/RecPointsQcTask.h

```diff
@@ -0,0 +1,115 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file   DigitQcTask.h
+/// \author Artur Furs afurs@cern.ch
+/// modified by Sebastin Bysiak sbysiak@cern.ch
+/// QC Task for FT0 detector, mostly for data visualisation during FEE tests
+
+#ifndef QC_MODULE_FT0_FT0RECOQCTASK_H
+#define QC_MODULE_FT0_FT0RECOQCTASK_H
+
+#include <Framework/InputRecord.h>
+
+#include "QualityControl/QcInfoLogger.h"
+#include "FT0Base/Geometry.h"
+#include "DataFormatsFT0/RecPoints.h"
+#include "DataFormatsFT0/ChannelData.h"
+#include "QualityControl/TaskInterface.h"
+#include <memory>
+#include <regex>
+#include <type_traits>
+#include <boost/algorithm/string.hpp>
+#include "TH1.h"
+#include "TH2.h"
+#include "TList.h"
+#include "Rtypes.h"
```

Comment:
Please use <> brackets for headers which are not in the QC project to speed up the compilation.